_posts/2017-06-03-gsoc-students.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Google Summer of Code Students 2017
+---
+
+We are happy to anounce that MDAnalysis is hosting a [GSoC][gsoc] students for
+the [NumFOCUS][numfocus] this year, [Utkarsh Bansal][utkarsh-gsoc].
```

Comment:
I don't like having the name link to the project description, I'd rather do something like

     ... this year, [Utkarsh Bansal](http://utkarshbansal.me/about/) (@utkbansal on GitHub), with his project ["Port to pytest"][utkarsh-gsoc].

so any link to his name points to something related to him as a person and it is also clear what his project is.